scripts/release_management/sha-files.py

```diff
@@ -1,35 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Does goreleaser generate these hashes now?

Reply:
yes, within .goreleaser.yml there is a checksum section.

Reply:
Can't comment on this specif question - though the reproducible buildsystem, if used, would certainly generate hashes